Tools/Testing/CMakeLists.txt

```diff
@@ -83,14 +83,17 @@ file(GLOB TM_CODE
 set(DRIVERS_FOLDER "${ROOT_DIR}/Drivers")
 set(DRIVERS_INC "${DRIVERS_FOLDER}/Common/Inc")
 set(DRIVERS_SRC "{$DRIVERS_FOLDER}/Src")
+set(MOTORCHANNEL_INC "${DRIVERS_FOLDER}/MotorChannel/Inc")
+set(MOTORCHANNEL_SRC "${DRIVERS_FOLDER}/MotorChannel/Src")
 
-include_directories(${DRIVERS_INC})
+include_directories(${DRIVERS_INC} ${MOTORCHANNEL_INC})
 file(GLOB DRIVERS_CODE
     "${DRIVERS_FOLDER}/Tests/*.cpp"
 
     # add necessary Drivers source files below
     # "${DRIVERS_SRC}/foo.cpp"
     "${DRIVERS_FOLDER}/Common/Src/circular_buffer.cpp"
+    "${MOTORCHANNEL_SRC}/*.cpp"
```

Comment:
You can fix the build issues you mentioned by either specifying the mock you wish to build, or creating a mock folder